includes/admin.form.inc

```diff
@@ -22,6 +22,12 @@ function islandora_batch_admin_form($form, $form_state) {
     '#description' => t('When a batch finishes, we will attempt to render a link to review the queue for the given set. This is dependent on the "views" module being enabled.'),
     '#default_value' => variable_get('islandora_batch_link_when_finished', TRUE),
   );
+  $form['islandora_batch_auto_remove_set'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Auto-remove batch set when processing finishes?'),
+    '#description' => t('When a batch set is processed, it will be removed from storage.'),
```

Comment:
I agree, I was kind of looking forward to a comment like this because I was unsure on how to word it. I'll update this in a bit.